IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs

```diff
@@ -50,6 +50,7 @@ public partial class AnyBitmap : IDisposable
         private Image Image { get; set; }
         private byte[] Binary { get; set; }
         private IImageFormat Format { get; set; }
+        private TiffCompression TiffCompression { get; set; } = TiffCompression.Lzw;
```

Comment:
Why is LZW set to be the default compression? Does it usually result in the smallest file size?

Reply:
LZW is widely used, offers good compression for various image types, especially those with repetitive patterns. Relatively fast for both compression and decompression. But not sure it the smallest file size.